.github/workflows/CI.yml

```diff
@@ -29,13 +29,13 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        node-version: [ 16 ]
+        node-version: [ 22 ]
```

Comment:
if we require minimum v20, shouldn't the test run on it to make sure that we deliver our promise of it working with Node@20?

Reply:
We test against stable (current) in the first build step: https://github.com/bpmn-io/bpmnlint/pull/149/files#diff-3ab46ee209a127470fce3c2cf106b1a1dbadbb929a4b5b13656a4bc4ce19c0b8R4.